### PR TITLE
Update release-notes-integration-services-2022.md

### PR DESCRIPTION
**Pull request by MSraye**

### CODE REVIEW

**Comment by Court72** [comment]
Close/reopen to clear validation warnings.

**Comment by Court72** [comment]
@chugugrace

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by chugugrace — APPROVED** [review]

**Comment by chugugrace** [comment]
#sign-off

File: docs/ssdt/release-notes-integration-services-2022.md [hunk 1]
```diff
@@ -33,15 +33,16 @@ Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-
   1. Side by side, localization and globalization aren't supported.
   1. Packages using Oracle and Teradata connectors aren't supported. 
   1. Azure-enabled SSIS projects aren't supported.
-
+  1. Right click Package and select SSIS Import and Export Wizard throws error, workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\DTSWizard.exe
+  1. Right click Package and select Upgrade All Packages throws error, workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\SSISUpgrade.exe
```
Comment (Court72) [inline docs/ssdt/release-notes-integration-services-2022.md:37]:
```suggestion
  1. Right-click Package and select SSIS Import and Export Wizard throws error. Workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\DTSWizard.exe
  1. Right-click Package and select Upgrade All Packages throws error. Workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\SSISUpgrade.exe
```

File: docs/ssdt/release-notes-integration-services-2022.md [hunk 2]
```diff
@@ -33,15 +33,16 @@ Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-
   1. Side by side, localization and globalization aren't supported.
   1. Packages using Oracle and Teradata connectors aren't supported. 
   1. Azure-enabled SSIS projects aren't supported.
-
+  1. Right click Package and select SSIS Import and Export Wizard throws error, workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\DTSWizard.exe
```
Comment (Court72) [inline docs/ssdt/release-notes-integration-services-2022.md:36]:
```suggestion
  1. Right-click Package and select SSIS Import and Export Wizard throws error. Workaround: execute Common7\IDE\CommonExtensions\Microsoft\SSIS\160\Binn\DTSWizard.exe
```

**Pull request merged**